papis/commands/default.py

```diff
@@ -121,6 +122,30 @@ def _on_finish() -> None:
     return _on_finish
 
 
+class LibType(click.ParamType):
```

Comment:
```suggestion
class LibraryParamType(click.ParamType):
```
? Also, maybe put this in `papis.cli`?